jobs/create-aci-db-new/templates/pre-start.ctl.erb

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+DB_NAME=<%= p('database.name') %>
+DB_USERNAME=<%= p('database.username') %>
+DB_PASSWORD=<%= p('database.password') %>
+DB_CONFIG_FILE="/var/vcap/jobs/mysql/config/mariadb_ctl_config.yml"
+
+cat > /tmp/data.txt <<EOF
```

Comment:
probably best to not write sensitive information to random files on the filesystem. Can't the replacement done inline without using another file?